Core/UI/Data/DataAggregator.hpp

```diff
@@ -23,18 +24,19 @@ using namespace std;
  */
 class DataAggregator {
 public:
-    explicit DataAggregator(uint8_t lapEfficiencySize): lapEfficiencies({BarDataCollection<watt_hour_t>(lapEfficiencySize)}) {
-
-    }
-
+    explicit DataAggregator(uint8_t motorVelocitiesSize, uint8_t batteryVoltagesSize, uint8_t lapEfficienciesSize, uint8_t lapTimesSize):
+            motorVelocities(motorVelocitiesSize),
+            batteryVoltages(batteryVoltagesSize),
+            lapEfficiencies(lapEfficienciesSize),
+            lapTimes(lapTimesSize) {}
     /** The observed object that holds the motor RPM data. */
-    ObservedObject<velocity_t> motorRPM{0};
```

Comment:
All of the observed objects are now queues cause we want to be able to track all of them.